### DISCUSSION TITLE
Google Play のデータ セーフティ セクションへの記載について

### DISCUSSION THREAD

**Discussion by hiirohitoyo** [pixiv/vroid-sdk-developers#31] [Q&A] [answered]
今年の4月以降、Google Playにてアプリを公開する際には、データセーフティについての情報提供が必要になるようです。
参考：https://support.google.com/googleplay/android-developer/answer/10787469?hl=ja

そのため、AndroidアプリでVRoid SDKを利用する際にどのように回答するのが適切か教えていただけないでしょうか。
アプリによって異なる部分はあるとは思いますが、例えば、個人識別情報（VRoid Hubアカウント）が収集・共有され、アプリの機能（VRMモデルのダウンロード・表示）に使われる、という回答はしなければならないのではないかと考えております。
特に、用途に「分析」「広告、マーケティング」があり、アプリ側では特にこれらの用途には使っていない場合でも、VRoid SDK/Hub側で使っている場合には、その用途で使っていると回答しなければいけないと認識しております。

加えて、データの転送時（ログイン時などが該当すると想定）に暗号化がなされているかどうかも回答しなければいけませんので、暗号化されているかどうかも教えていただけますと幸いです。

**Comment by Haresaku** [comment]
ご連絡ありがとうございます。こちら一度調査させていただきますのでしばらくお待ちください

**Comment by Haresaku** [comment, marked as answer]
@hiirohitoyo こちら確認させていただきました。データセーフティについての情報共有ですが、VRoid Hub/SDK側では「分析」の用途でデータを利用している、転送時の暗号化についてはSSLでのリクエストの暗号化を行っているのみ、と回答をしていただけますでしょうか。

**Reply by hiirohitoyo** [reply]
ご回答ありがとうございます！
では、「分析」の用途でデータを利用、転送時の暗号化についてはSSLでのリクエストの暗号化のみ、ということで回答させていただきます。返信遅くなりましてすみません。
